.github/workflows/build-docs-dev.yml

```diff
@@ -46,3 +46,5 @@ jobs:
         uses: JamesIves/github-pages-deploy-action@v4
         with:
           folder: _build
+          target-folder: main
```

Comment:
I think there should be `target-folder: master` because this repo has an old "master" name for its main branch. Do we have any convention regarding this?

Reply:
This target folder doesn't need to be named `master` and to keep it consistent across projects I'm using `main`

Reply:
Ok, thank you